test/e2e/page-objects/pages/home/nft-list.ts

```diff
@@ -6,11 +6,20 @@ class NftListPage {
   private readonly confirmImportNftButton =
     '[data-testid="import-nfts-modal-import-button"]';
 
+  private readonly importNftNetworkDropdown =
+    '[data-testid="test-import-tokens-drop-down-custom-import"]';
+
+  private readonly importNftNetworkName =
+    '[data-testid="select-network-item-0x539"]';
```

Comment:
This should be passed in as a prop to the e2e test. Will update this to test for both cases:
1. Adding NFT on network that I own an NFT (should pass)
2. Adding NFT on network that I don't own (should fail)